apps/web/components/Proposals.tsx

```diff
@@ -368,6 +363,29 @@ export function Proposals({
     }
   };
 
+  const poolWeightClassName = `bg-primary-soft text-primary-content ${calcPoolWeightUsed(memberSupportedProposalsPct) == memberPoolWeight && "bg-secondary-soft text-secondary-content"}`;
+
+  const stats = [
+    {
+      id: 1,
+      name: "Pool Weight",
+      stat: memberPoolWeight,
+      className: poolWeightClassName,
+    },
+    {
+      id: 2,
+      name: "Allocated Pool Weight",
+      stat: calcPoolWeightUsed(memberSupportedProposalsPct),
+      className: poolWeightClassName,
+    },
+    {
+      id: 3,
+      name: "Total Allocation Percentage",
+      stat: memberSupportedProposalsPct,
+      className: `bg-primary-content text-primary-soft border-primary-content ${memberSupportedProposalsPct >= 100 && "bg-secondary-content text-secondary-soft border-secondary-content "}`,
```

Comment:
I know its very sad but you cant do that without printing false in the classnames :S
![image](https://github.com/1Hive/gardens-v2/assets/13890618/0cf2bb3c-840f-4a16-a3c1-02a08e3f70ac)